### PR TITLE
change doc to recommend metals worksheet

### PR DESCRIPTION
**Pull request by bishabosha**

### CODE REVIEW

**Comment by smarter** [comment]
Seems like we could just delete that page and point to the metals documentation which will always be more up to date

File: docs/docs/usage/worksheet-mode.md [hunk 1]
```diff
@@ -24,27 +25,20 @@ println("Hello, worksheets!")
 456 + xyz
 ```
 
-On top of the buffer, the message `Run this worksheet` appears. Click it to
-evaluate the code of the worksheet. Each line of output is printed on the right
+On top of the buffer, the message `Copy Worksheet Output` appears. Save the file to evaluate the code of the worksheet. Each line of output is printed on the right
```
Comment (tgodzik) [inline docs/docs/usage/worksheet-mode.md:28]: I wouldn't mention `Copy Worksheet Output` here, it can be mentioned additionally.

**Review by tgodzik — CHANGES_REQUESTED** [review]
@smarter might be right that Metals worksheet docs should be always more up to date. Is the current state of the docs enough though? https://scalameta.org/metals/docs/editors/vscode.html#worksheets

**Comment by bishabosha** [comment]
I will close do to the other rewrite being proposed by https://github.com/scala/docs.scala-lang/pull/2112

**Pull request closed without merging**